__tests__/index.test.js

```diff
@@ -315,7 +315,21 @@ describe("Purchases", () => {
 
     expect(NativeModules.RNPurchases.setDebugLogsEnabled).toBeCalledWith(false);
     expect(NativeModules.RNPurchases.setDebugLogsEnabled).toBeCalledTimes(2);
-  })
+  });
+
+  it("setLogLevel works", () => {
+    const Purchases = require("../dist/index").default;
+
+    Purchases.setLogLevel(Purchases.LOG_LEVEL.ERROR);
+
+    expect(NativeModules.RNPurchases.setLogLevel).toBeCalledWith("ERROR");
+    expect(NativeModules.RNPurchases.setLogLevel).toBeCalledTimes(1);
+
+    Purchases.setLogLevel(Purchases.LOG_LEVEL.VERBOSE);
+
+    expect(NativeModules.RNPurchases.setLogLevel).toBeCalledWith("VERBOSE");
+    expect(NativeModules.RNPurchases.setLogLevel).toBeCalledTimes(2);
+  });
```

Comment:
Probably too much, but I'd like to mention in case it wasn't intentional. This is only testing for Error and Verbose.

Reply:
Good point. I added tests for all levels.